### PR TITLE
LPS-182598 | Modify added message template header

### PR DESCRIPTION
**Pull request by jaimelr10**
**Description:** Follow up to https://github.com/liferay-echo/liferay-portal/pull/12249 in order to correct a small mistake on the added message template.

**Jira Ticket:** https://issues.liferay.com/browse/LPS-186152

### CODE REVIEW

File: portal-impl/src/com/liferay/portlet/messageboards/dependencies/email_message_added_body_LPS-182020.tmpl [hunk 1]
```diff
@@ -171,7 +171,7 @@
 </style>
 
 <div class="mb-mail-message">
-	<p>There is <strong>1 new message </strong>in the thread.</p>
+	<p>There is <strong>1 new message </strong>.</p>
```
Comment (marcogalluzzi) [inline portal-impl/src/com/liferay/portlet/messageboards/dependencies/email_message_added_body_LPS-182020.tmpl:174]: Please remove empty space at the end

**Review by marcogalluzzi — CHANGES_REQUESTED** [review]

**Comment by victorg1991** [comment]
👏

**Comment by victorg1991** [comment]
ci:forward

**Review by marcogalluzzi — APPROVED** [review]

**Comment by Tim-Cao** [comment]
ci:forward:force

**Comment by Tim-Cao** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/135915

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**